app/views/issues/new.slim

```diff
@@ -0,0 +1,31 @@
+h1 Create new issue
+= form_for @issue, url: issue_path do |f|
+  - if @issue && @issue.errors.any?
+    div.error
+      h2 =pluralize(@issue.errors.count, 'error')
+      | prohibited this issue from being saved:
+      ol
+        - @issue.errors.full_messages.each do |msg|
+          li =msg
+  = f.label :name
+  = f.text_field :name
+  br/
+  = f.label :address
+  = f.text_field :address
+  br/
+  = f.label :phone
+  = f.text_field :phone
+  br/
+  = f.label :email
+  = f.email_field :email
+  br/
+  = f.label :category
+  = f.text_field :category
+  br/
+  = f.label :description
+  = f.text_area :description 
+  br/
+  = f.label :attachment
```

Comment:
Please, implement attachment fictionality here.

> Created form for creating issues

It means that we should have totally working solution, not 80% but 100%.  It is pretty easy to implement. Use [carrierwave](https://github.com/carrierwaveuploader/carrierwave) for it. 

In case project will be on heroku you should create and assign to someone or to yourself issue "Upload all assets to Amazon S3".